httpmw/logging/logging_test.go

```diff
@@ -14,24 +14,26 @@ import (
 
 var _ = Describe("Httpmw", func() {
 	var (
-		router *mux.Router
+		router     *mux.Router
+		buf        *bytes.Buffer
+		slogLogger *slog.Logger
 	)
 
 	BeforeEach(func() {
+		buf = new(bytes.Buffer)
+		slogLogger = slog.New(slog.NewJSONHandler(buf, nil))
+
 		router = mux.NewRouter()
+		router.Use(NewLogging(slogLogger))
```

Comment:
Add a test case that use both middleware.

Reply:
done